modules/Layout/Branding/utils.ts

```diff
@@ -14,47 +15,17 @@ function getFontFamily(font: Font): string {
     return FONT_FAMILY[font] || Font.INTER;
 }
 
-function getDefaultCssVariables() {
-    const accentColor = '#3b82f6';
-    const accentColorButtonText = '#ffffff';
-    const headerBackgroundColor = '#ffffff';
-    const headerLinkColor = '#374151';
-    const placeholderBackgroundColor = '#f3f4f6';
-
-    return [
-        `--prezly-font-family: ${Font.INTER}`,
-        `--prezly-accent-color: ${accentColor}`,
-        `--prezly-accent-color-tint: ${tinycolor(accentColor)
-            .lighten(ACCENT_COLOR_TINT_FACTOR)
-            .toHexString()}`,
-        `--prezly-accent-color-shade: ${tinycolor(accentColor)
-            .darken(ACCENT_COLOR_SHADE_FACTOR)
-            .toHexString()}`,
-        `--prezly-accent-color-button-text: ${accentColorButtonText}`,
-        `--prezly-header-background-color: ${headerBackgroundColor}`,
-        `--prezly-header-link-color: ${headerLinkColor}`,
-        `--prezly-placeholder-background-color: ${placeholderBackgroundColor}`,
-    ];
-}
-
-export function getCssVariables(themePreset: NewsroomThemePreset | null) {
-    if (!themePreset) {
-        return getDefaultCssVariables();
-    }
-
+export function getCssVariables(themeSettings: ThemeSettings) {
     const {
         accent_color: accentColor,
         font,
         header_background_color: headerBackgroundColor,
         header_link_color: headerLinkColor,
-    } = themePreset.settings as ThemeSettings;
+    } = themeSettings;
 
     // Use the default placeholder color if the header background color has not been changed
     const placeholderBackgroundColor =
-        // TODO: Check the typings and fix it.
-        // `default` property is missing from typings but is available in the API
-        // @ts-expect-error
-        themePreset.editable_settings.properties.header_background_color?.default.toLowerCase() ===
+        DEFAULT_THEME_SETTINGS.header_background_color.toLowerCase() ===
```

Comment:
Since we already have hardcoded default values for Bea I've updated this part and got rid of the TS error.